### ISSUE TITLE
Route statamic.password.reset not found

### ISSUE THREAD

**Issue by crackAT** [statamic/cms#3444]
## Bug Description
Password rest route not found when i try to send a password reset.
## How to Reproduce
When use "Send Passwort Reset" under cp/users i will get the Route error. 

Maybe the route name should be statamic.cp.password.reset? 

## Extra Detail
![image](https://user-images.githubusercontent.com/6299060/112987714-517f0200-9163-11eb-9673-72967df04ad7.png)
![image](https://user-images.githubusercontent.com/6299060/112987752-593ea680-9163-11eb-98b7-61d8424001f9.png)
![image](https://user-images.githubusercontent.com/6299060/112987786-5f348780-9163-11eb-9401-80235a1fff65.png)

I'm using Database Users. 
## Environment

Statamic 3.1.1 Pro
Laravel 8.34.0
PHP 7.3.11
4rn0/statamic-v3-image-optimizer 1.0.7
aryehraber/statamic-impersonator 2.2.0
aryehraber/statamic-logbook 2.0.1
phpsa/statamic-analytics 1.3.0
rbmh/statamic-filtered-terms-fieldtype 1.0.2
rbmh/statamic-location-fieldtype 1.0.3
rbmh/statamic-redirects 1.0.3
statamic/eloquent-driver 0.1.1

- Existing Laravel app

**Comment by jasonvarga** [comment]
Do you have routes disabled in config/statamic/routes.php ?

**Comment by crackAT** [comment]
Yes i'm using statamic headless. Is there a workaround, adding the routenames to my web routes?

**Comment by jasonvarga** [comment]
Huh. I'm thinking maybe we disable too much with that setting. Looks like Glide wouldn't work either.

Anyway, for now you can work around it by adding the routes to your own `routes/web.php` file manually.

```php
Route::post('!/password/email', 'ForgotPasswordController@sendResetLinkEmail')->name('statamic.password.email');
Route::get('!/password/reset/{token}', 'ResetPasswordController@showResetForm')->name('statamic.password.reset');
Route::post('!/password/reset', 'ResetPasswordController@reset')->name('statamic.password.reset.action');
```
